Gemfile.lock

```diff
@@ -0,0 +1,57 @@
+PATH
```

Comment:
Please leave me questions or feedback anywhere on this pull request!

I'd also be curious to learn anything I'm doing that is no longer recommended, there is an alternative you'd recommend, or can otherwise be improved!